node_modules/oae-core/tenantlandingpage/tenantlandingpage.html

```diff
@@ -0,0 +1,88 @@
+<!-- CSS -->
+<link rel="stylesheet" type="text/css" href="css/tenantlandingpage.css" />
+
+<!-- TENANT LANDING PAGE -->
+<div id="tenantlandingpage"><!-- --></div>
```

Comment:
Shouldn't use an `id` here anymore as widgets can theoretically be used multiple times in the same page. Should therefore be converted to a class